examples-spark/src/main/resources/sample_executor_resources.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
This duplicates the one in examples-beam. Can you share code?

Reply:
Refactored.